dvc/info.py

```diff
@@ -134,3 +142,26 @@ def _get_dvc_repo_info(self):
         return "dvc (subdir), git"
 
     return "dvc, git"
+
+
+def _get_external_remotes_cache(self):
+    cache_info = self.config.get("cache", {})
+
+    cache_vars = [
+        val for val in Schemes.__dict__.values() if isinstance(val, str)
+    ]
+    cache_key = list(set(cache_info.keys()).intersection(set(cache_vars)))
+
+    core_info = self.config.get("core", {})
+    cache_location, remote_url = "", ""
+
+    if cache_info and cache_key:
+        cache_location = cache_info[cache_key[0]]
```

Comment:
A repo might have multiple caches (e.g. one for s3, gs, etc). The original ticket mostly focuses on local external cache dir (yeah, I bit confusing, I know :slightly_smiling_face: ), so `local` or `dir` keys in `cache` config section.

Reply:
Yes I agree but won't _cache_info_ show the current cache in use because I assume you can't use multiple caches at the same time or am I wrong on this?

Reply:
You can, see https://dvc.org/doc/user-guide/managing-external-data . It is a rather confusing part of dvc subsystems :slightly_frowning_face: In this case, I would say that @skshetry meant only `local` and `dir`, as those are the most useful local cache dir config options that many people use. The rest are used by a few people in a very advanced scenarios, and we could totally just ignore those :slightly_smiling_face:

Reply:
@efiop, I meant all kinds of cache types in the dvc version or at least in the log during `--debug`, see the discord discussion for the motivation: https://discord.com/channels/485586884165107732/485596304961962003/755067231339020358.

Reply:
@sanketsaurav all kinds of caches should now show up in the output of `dvc version` command.